### DISCUSSION TITLE
可以修改一下课程名称？

### DISCUSSION THREAD

**Discussion by depetrol** [Yang-Xijie/yang-xijie.github.io#5] [Cpp in VSCode on macOS]
现在b站上的课程名称有三个限定词，macos，vscode，c++，但是看完之后觉得这个教程其实涵盖的远远不止这些，可以修改一下课程的名称，也许能让更多人看到这样优秀的教程。

**Comment by Yang-Xijie** [comment]
如果可以能告诉我一些你想到的关键词吗？我之后考虑添加😃

**Reply by depetrol** [reply]
我也想不太到耶》》比如说命令行？我的意思是限定词太多了反而会没什么人看，比如说我是一个windows用户就不会点开这个视频。我自己瞎想的一个标题《大学没教的程序设计：从命令行到vscode》hhh

**Reply by Yang-Xijie** [reply]
其实我这个教程还是主要面向macOS，C/Cpp代码编写的同学的，这几个关键词也足够。标题每节课也不太一样，有的课我也贴了不一样的标签，应该能让不少同学搜到。主要还是推广课程吧，让更多人看到。从视频上传到现在也挺多好评的，以后看有没有更多的途径去推广。如果以后有更好的标题也会考虑更改哦～谢谢支持！😉

**Reply by depetrol** [reply]
好的加油~视频真的太棒了！
